src/app/payments/container.tsx

```diff
@@ -1,37 +1,40 @@
+import { WalletAddress } from 'electra-js'
 import * as React from 'react'
-import { DispatchProps, State, State as Props } from './types'
-const { connect } = require('react-redux')
-import { get } from 'lodash'
+import { connect, MapDispatchToProps, MapStateToProps } from 'react-redux'
 import { bindActionCreators, Dispatch } from 'redux'
-
 import { setMessageAndBadge } from '../common/toast/actions'
 import { COPIED_ADDRESS, PENDING, SENDING_IN_PROGRESS, SUCCESS } from '../common/toast/toast-messages'
-import { clearSendCardFields, sendEca, setAmount, setToAddress } from './actions'
+import { clearSendCardFields, getAddresses, sendEca, setAmount, setToAddress } from './actions'
 import ReceiveCardView from './components/receive-card-view'
 import SendCardView from './components/send-card-view'
+import { DispatchProps, StateProps } from './types'
 
 // tslint:disable-next-line:typedef
-const mapStateToProps = (state: State): Props =>
-  ({
-    addresses: get(state, 'electra.electraJs.wallet.ADDRESSES', []),
-    payments: state.payments
-  })
+const mapStateToProps: MapStateToProps<StateProps,{}, {}> = (state: StateProps): StateProps => ({
+  payments: state.payments
+})
 
 // tslint:disable-next-line:typedef
-const mapDispatchToProps = (dispatch: Dispatch<{}>): DispatchProps =>
+const mapDispatchToProps: MapDispatchToProps<DispatchProps, {}> =
+(dispatch: Dispatch<StateProps>): DispatchProps =>
   bindActionCreators({
     clearSendCardFields,
+    getAddresses,
     sendEca,
     setAmount,
     setMessageAndBadge,
     setToAddress
     // tslint:disable-next-line:align
   }, dispatch)
 
-@connect(mapStateToProps, mapDispatchToProps)
-export default class Payments extends React.Component<Props & DispatchProps, any> {
+class Payments extends React.Component<StateProps & DispatchProps, any> {
```

Comment:
Remove the tslint:disable-next-line:typedef and tslint: disable-next-line:align.

mapDispatchToProps line aligned can be fixed if you replace it by:

```
const mapDispatchToProps: MapDispatchToProps<DispatchProps, {}> =
(dispatch: Dispatch<StateProps>): DispatchProps =>
  bindActionCreators
  (
    {
      clearSendCardFields,
      getAddresses,
      sendEca,
      setAmount,
      setMessageAndBadge,
      setToAddress
    },
    dispatch
  )
```